docs/fundamentals/toc.yml

```diff
@@ -149,8 +149,12 @@ items:
     href: ../standard/whats-new/whats-new-in-dotnet-standard.md
 - name: Tools and diagnostics
   items:
-  - name: .NET Core SDK overview
-    href: ../core/sdk.md
+  - name: .NET Core SDK
+    items:
+    - name: Overview
+      href: ../core/sdk.md
+    - name: Error messages
+      href: ../core/tools/netsdk1145.md
```

Comment:
This link is to the wrong file now. But thinking about it, I think the current path of the file (_core/tools/sdk-errors_) is too deep. I think we should move it to _core/sdk-errors/_

Thoughts on that @tdykstra ?